static/js/around-map.js

```diff
@@ -46,20 +45,26 @@ worker.addEventListener('message', response => {
 worker.postMessage('')
 
 // event
+window.onhashchange = _ => {
+  const hash = getHash()
+  masterCard(hash)
+}
+
+window.onload = _ => {
+  // section poisCards hidden marker overflow
+  const canvas = document.querySelector('canvas')
+  const poisCards = document.querySelector('#poi-cards')
+  poisCards.style.minHeight = `${canvas.height}px`
```

Comment:
because I want the `min-height` change according the map. I use `poisCards` to hidden marker when they are outside canvas. So I want know how many pixel the map have